gradle-repository/buildSrc/src/main/kotlin/gradlebuild.gradle-api-repository.gradle.kts

```diff
@@ -0,0 +1,85 @@
+plugins {
+    base
+    `maven-publish`
+}
+
+val gradleApiRepository = extensions.create<GradleRepositoryExtension>("gradleApiRepository")
+
+gradleApiRepository.apply {
+    gradleVersion.convention(
+        providers.gradleProperty("gradleApiVersion").orElse("7.4.1")
```

Comment:
Why `7.4.1` specifically? Why not the current Gradle version?

Reply:
You mean the current version from the wrapper? I don't feel strongly what the convention should be. Do you think it makes sense to force the user to provide a version via a Gradle property and not have any default?

Reply:
Forcing the user sounds like the right way to go here, no? Does it mean anything to say "publish a Gradle API so I can compile against it, I don't care which version?"

Reply:
I removed the default.